lib/hexpm_web/controllers/dashboard/organization_controller.ex

```diff
@@ -368,6 +368,77 @@ defmodule HexpmWeb.Dashboard.OrganizationController do
     end)
   end
 
+  def add_email(conn, %{"dashboard_org" => organization, "email" => email_params}) do
+    access_organization(conn, organization, "admin", fn organization ->
+      case Users.add_email(organization.user, email_params, audit: audit_data(conn)) do
+        {:ok, _user} ->
+          conn
+          |> put_flash(:info, "A verification email has been sent to #{email_params["email"]}.")
+          |> redirect(to: Routes.organization_path(conn, :show, organization))
+
+        {:error, _changset} ->
+          conn
+          |> put_flash(:error, "Oops, something went wrong!")
```

Comment:
I used `Oops, something went wrong!` for all the errors because I was not sure how to reuse `EmailController.email_error_message/2`
To me, this error message translation seems to be an I18n concern. But we are not using any I18n libraries for now, right?

A few options:
1. make `email_error_message/2` public and call it directly here
2. move `email_error_message/2` to `EmailView` and call it here
3. copy/paste `email_error_message/2` to this controller

Do you have any suggestions?